packages/fxa-graphql-api/src/lib/auth.ts

```diff
@@ -13,10 +13,9 @@ export class SessionTokenAuth {
   @Inject(fxAccountClientToken)
   private authClient!: AuthClient;
 
-  public async lookupUserId(sessionToken: string): Promise<string> {
+  public async lookupUserId(sessionToken: string) {
     try {
-      const result = await this.authClient.sessionStatus(sessionToken);
-      return result.uid;
```

Comment:
instead of keeping just the `uid` by keeping the `state` we can use it later. much of the downstream changes are just to handle the object instead of string